.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+# Default ignored files
+/workspace.xml
```

Comment:
Please add your `.idea` directory to a global gitignore once and for all: https://greg0ire.fr/git-gud/#/23

Reply:
Sorry about that. :pray: